deploy/csi-operator.yaml

```diff
@@ -497,14 +491,29 @@ status:
   storedVersions: []
 
 ---
+
+apiVersion: storage.k8s.io/v1beta1
+kind: CSIDriver
+metadata:
+  name: cstor.csi.openebs.io
+spec:
+  # Supports persistent inline volumes.
+  volumeLifecycleModes:
+  - Persistent
+  # To determine at runtime which mode a volume uses, pod info and its
+  # "csi.storage.k8s.io/ephemeral" entry are needed.
+  podInfoOnMount: true
+  attachRequired: true
+---
```

Comment:
@kmova added the CSIDriver volumeLifecycle Mode as well since we are setting up k8s 1.17.x as minimum version

Reply:
@prateekpandey14  Does this mean the minimum version of k8s for openebs from 1.12 release should be 1.17?

Reply:
Not for openebs overall, its specific to CSI/CSPC with all the feature supported like migration/upgrade etc.